lib/routes/devices-and-sessions.js

```diff
@@ -237,6 +243,94 @@ module.exports = (log, db, config, customs, push, devices) => {
         }
       }
     },
+    {
+      method: 'GET',
+      path: '/account/device/messages',
+      config: {
+        validate: {
+          query: {
+            service: isA.string().valid('sendtab').required(),
+            index: isA.string().optional(),
```

Comment:
can we use any existing validator to tighten the allowed values here? People keep fuzzing our API

Reply:
Pushbox might change the way it represents the indexes, that's why we have a broad definition for limit.

Reply:
IMHO it's not safe to allow any arbitrary string here, we've had a number of close calls due to unexpected characters being allowed through into URL paths and params.  Can we make it e.g. alphanumberic or urlsafe-base64 or something?  

`URLSearchParams.toString()` should guard against any weirdness by properly escaping things in the current implementation, but `index` sounds so innocuous that future devs might not be so careful.

(I also think it's reasonable to push the pushbox team to commit to a set of allowed characters here; if they want to be super flexible then can always b64-encode values on the way out and decode then again on the way in).